Equilobe.DailyReport.SL/DataService.cs

```diff
@@ -440,6 +440,19 @@ public TimeSpan GetOffsetFromProjectKey(string key)
             return TimeZoneHelpers.GetOffsetFromTimezoneId(timeZoneId);
         }
 
+        public TimeSpan GetOffsetFromInstanceId(long instanceId)
+        {
+            using (var db = new ReportsDb())
+            {
+                var timeZoneId = db.InstalledInstances
+                    .Where(p => p.Id == instanceId)
```

Comment:
Same here. Do not use where. Use First or Single instead. We expect a single instance not a list.